openfisca_uk/variables/demographic/person.py

```diff
@@ -18,14 +18,39 @@ class people(Variable):
     default_value = 1
 
 
-class person_weight(Variable):
+class FRS_person_weight(Variable):
     value_type = float
     entity = Person
-    label = u"Weight factor for the person"
+    label = "Weight factor"
+    documentation = "Used to translate from survey respondents to UK residents"
     definition_period = YEAR
     default_value = 1
 
 
+class person_weight(Variable):
+    value_type = float
+    entity = Person
+    label = "Weight (region-adjusted)"
+    documentation = "Adjusted to match regional population estimates"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        nation = person.household("country", period)
+        national_population = parameters(
+            period
+        ).demographic.population_estimate[nation]
+        national_weight_sum = (
```

Comment:
Suggest adding `frs_person_weight = person("FRS_person_weight", period)` to use in the two places it's called.